### PR TITLE
fix(dsl): Add support for files in dsl

### PR DESCRIPTION
**Pull request by sindrig**
This closes island-is/infrastructure#923

### CODE REVIEW

File: apps/api/infra/api.ts [hunk 1]
```diff
@@ -161,6 +161,7 @@ export const serviceSetup = (services: {
       Education,
       NationalRegistry,
     )
+    .files('islyklar.p12')
```
Comment (pshomov) [inline apps/api/infra/api.ts:164]: They usually need an env variable with the path to the file similar to [this](https://github.com/island-is/helm/blob/main/charts/identity-server/values.yaml#L30). Make this a hash where the key is the env variable name and the file is the value or the other way around 🙃

Comment (sindrig) [reply apps/api/infra/api.ts:164]: fair enough

Comment (sindrig) [reply apps/api/infra/api.ts]: done

**Review by pshomov — APPROVED** [review]
LGTM

**Pull request merged**